custom_types/src/lib.rs

```diff
@@ -33,7 +33,10 @@ impl IncrementContract {
     pub fn get_state(env: Env) -> State {
         env.storage()
             .get(&STATE)
-            .unwrap_or_else(|| Ok(State::default())) // If no value set, assume 0.
+            .unwrap_or(Ok(State {
+                count: 0,
+                last_incr: 0,
+            })) // If no value set, assume 0.
```

Comment:
I'm surprised I need to fix this in this PR as I thought I was just changing how tests ran in CI, but I think I might have in the process plugged a bug in the CI run that wasn't running all the builds / tests, because the custom_types crate is failing to build after I changed the CI workflow.